src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -721,8 +755,9 @@ function medlyn_conductance(
     medlyn_term::FT,
     An::FT,
     ca::FT,
+    β::FT,
 ) where {FT}
-    gs = g0 + Drel * medlyn_term * (An / ca)
+    gs = β * (g0 + Drel * medlyn_term * (An / ca))
```

Comment:
This should be: 

gs =  (g0 + β * Drel * medlyn_term * (An / ca))

Reply:
We should probably look at the plots together with this implemented. With this change, beta doesn't seem to have enough impact on GPP and we overestimate GPP for the entire year.

Reply:
@Yujie-W, could you please double check this equation?